share/completions/emaint.fish

```diff
@@ -1,32 +1,32 @@
 ## Global Opts
 complete -c emaint -s h -l help         -d "show this help message and exit"
-complete -c emaint -s c -l check        -d "Check for problems (a default option for most modules)"
+complete -c emaint -s c -l check        -d "check for problems"
```

Comment:
Ummmhh.. other way around, please!

It should be "Check for problems", not "check for problems".